package/config.js

```diff
@@ -36,5 +36,6 @@ const getPublicPath = () => {
 
 config.publicPath = getPublicPath()
 config.publicPathWithoutCDN = `/${config.public_output_path}/`
+config.cachePath = 'node_modules/.cache/webpacker'
```

Comment:
Thanks for the changes but I think this is handled internally if we set: `{cacheDirectory: true}` so no need to set this for loaders

https://webpack.js.org/loaders/babel-loader/#options